app/views/last_seen_mailer/last_seen_more_than_3_days.html.erb

```diff
@@ -1,102 +1,417 @@
+    background: #fff;
+    padding: 20px;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  }
+
+  .sticky-header {
+    top: 0;
+    border-bottom: 1px solid #ddd;
+  }
+
+  .sticky-footer {
+    bottom: 0;
+    border-top: 1px solid #ddd;
+  }
+
+  /* Scrollable Content Area */
+  .scrollable-content {
+    flex: 1;
+    overflow-y: auto;
+    padding: 100px 20px 120px; /* leave space for fixed header and footer */
+    background-color: #f9f9f9;
+    margin-top: 100px;
+    margin-bottom: 100px;
+  }
+
+  /* Optional: clean spacing for sections inside */
+  .district-section {
+    margin-bottom: 30px;
+  }
+
+  .site-table {
+    width: 100%;
+    border-collapse: collapse;
+    margin-top: 10px;
+  }
+
+  .site-table th, .site-table td {
+    border: 1px solid #ccc;
+    padding: 8px;
+    text-align: left;
+  }
+
+  .site-table th {
+    background-color: #eee;
+  }
+
+  .siteSearch {
+    width: 100%;
+    padding: 8px;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    font-size: 15px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+  }
+
+    /* Mobile responsiveness */
+    @media (max-width: 768px) {
       .container {
-        max-width: 600px;
-        background: #ffffff;
-        padding: 20px;
-        margin: auto;
-        border-radius: 8px;
-        box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.1);
+        padding: 20px 5%;
       }
-      h1 {
-        color: #6c757d;
-        font-size: 20px;
+
+      .title {
+        font-size: 22px;
       }
-      p {
-        font-size: 14px;
-        line-height: 1.6;
+
+      .intro,
+      .footer-text,
+      .signature {
+        font-size: 16px;
       }
-      table {
-        width: 100%;
-        border-collapse: collapse;
-        margin-top: 15px;
+
+      .district-heading {
+        font-size: 18px;
       }
+
       th, td {
-        border: 1px solid #ddd;
-        padding: 10px;
-        text-align: left;
-      }
-      th {
-        background: #6c757d;
-        color: white;
-        font-weight: bold;
-      }
-      tr:nth-child(even) {
-        background-color: #f9f9f9;
-      }
-      a {
-        color: #5a6268;
-        text-decoration: none;
-        font-weight: bold;
+        font-size: 14px;
+        padding: 8px;
       }
-      .footer {
-        font-size: 12px;
-        color: #777;
-        text-align: center;
-        margin-top: 20px;
+  
+      #districtSearch {
+        font-size: 15px;
       }
-    </style>
-  </head>
-  <body>
-    <div class="container">
-      <h1>Connectivity Issue Detected: No connectivity with <%= @district_data[:sites][:sites_last_seen_greater_than_3_days].count %> sites in <%= @district_data[:name] %>
-      </h1>
-      <p>  
-        Kindly be informed that these sites have been offline for  
-        <strong>three</strong> days, and no data has been received during this period.  
+    }
+  </style>
+</head>
+<body>
+   <div class="page-wrapper">
+    <!-- Sticky Header -->
+    <div class="sticky-header">
```

Comment:
This seems to be taking much more space maybe we can reduce is size a bit so that the contect has more space